### PR TITLE
Mptasks

### PR DESCRIPTION
**Pull request by Misha86**
tasks 86a 86b 330
tests 87 226 559

### CODE REVIEW

File: tasks/task_330.py [hunk 1]
```diff
@@ -1,20 +1,27 @@
 """This module provides function to get perfect numbers."""
 import time
```
Comment (morento101) [inline tasks/task_330.py]: from time import time

File: tests/test_task_226.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
+""" Group tests for tasks. """
+import unittest
+from tasks.task_226 import task_226
+
+
+class TestTask226(unittest.TestCase):
+    """
+    Tests for function task_226.
+    """
+
```
Comment (morento101) [inline tests/test_task_226.py]: Empty line

File: tests/test_task_559.py [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+    def test_valid(self):
+        """
+        Test that task_559 function returns correct result.
+        """
+        self.assertListEqual(task_559(10), [1, 3, 7])
+
+    def test_arg_incorrect_type(self):
+        """
+        Test that the arg has incorrect data type a TypeError exception is called.
```
Comment (morento101) [inline tests/test_task_559.py]: AssertionError

**Review by morento101 — CHANGES_REQUESTED** [review]
Change import, remove the empty line, replace TypeError with AssertionError in doc str, resolve conflicts

**Review by Misha86 — COMMENTED** [review]
corrected some files

**Pull request merged**